pod-operation/src/state_machine.rs

```diff
@@ -183,6 +186,17 @@ impl StateMachine {
 		self.brakes.engage();
 	}
 
+	fn _enter_faulted(&mut self) {
+		info!("Entering Faulted state");
+		self.io
+			.of("/control-station")
+			.unwrap()
+			.emit("fault", "123")
+			.ok();
+		self.signal_light.disable();
+		self.brakes.engage();
```

Comment:
Chore: Following from #79 (once that is merged), we'll need to also disable the high-voltage system.

Reply:
yuh